test/test_nn.py

```diff
@@ -14354,6 +14354,14 @@ def _make_noncontiguous(inp):
             bias.requires_grad_(False)
         self.assertTrue(gradgradcheck(convolution, inputs, nondet_tol=gradcheck_nondet_tol))
 
+
+    def test_channel_shuffle_runtime_errors(self, device):
```

Comment:
now that the behavior was changed to return self, I think this test name should be updated to match